src/involvement/cron.py

```diff
@@ -72,8 +72,8 @@ def send_extension_emails():
 def remove_old_applications():
     old_applications = Application.objects.filter(
         position__recruitment_end__lte=date.today() - timedelta(days=7)
-    ).exclude(
-        status='appointed'
+    ).filter(
+        status='turned_down'
```

Comment:
The change to remove only turned_down applications is good, because currently, it expects that applications are appointed within a week of the application end date, which is not always the case. The status is set to turned_down only when they have been appointed.